src/quest_manager/tests/test_views.py

```diff
@@ -159,6 +161,97 @@ def test_start(self):
         # the view should have redirect to the same submission:
         self.assertRedirects(response, sub.get_absolute_url())
 
+    @tag("do")
+    def test_student_no_quests_help_text(self):
+        """ 
+            When student has no quests but have:
+
+            quests in the In Progress Tab:
+                "You have no new quests available, but you can find some quests you have already started in your 'In Progress' tab above."
+
+            hidden quests:
+                "You have no new quests available, but you do have hidden quests which you can view by hitting the 'Show Hidden Quests' button above."
+
+            No in-progress and no hidden quests, but quests awaiting approval: 
+                "New quests will become available after your teacher approves your submission!"
+
+        """
+        url = reverse('quests:available')
+        user = self.test_student1
+
+        # login
+        success = self.client.login(username=user.username, password=self.test_password)
+        self.assertTrue(success)
+
+        # condition to see any of the 3
+        self.assertTrue(user.profile.has_current_course and not user.is_staff)
```

Comment:
Excellent comment.  I know it seems small and silly, but allowed me to instantly understand why you were asserting this.